starr/models.py

```diff
@@ -288,16 +274,12 @@ def guild(self) -> StarrGuild:
 
     async def db_insert(self, db: Database) -> None:
         await db.execute(
-            "INSERT INTO starboard_messages (StarMessageID, ReferenceID) "
-            "VALUES ($1, $2) ON CONFLICT DO NOTHING;",
+            "INSERT INTO starboard_messages (StarMessageID, ReferenceID) VALUES ($1, $2) ON CONFLICT DO NOTHING;",
```

Comment:
This has to be split, it fails the CI for len8.

```suggestion
            "INSERT INTO starboard_messages (StarMessageID, ReferenceID) "
            "VALUES ($1, $2) ON CONFLICT DO NOTHING;",
```

Reply:
committed locally